### DISCUSSION TITLE
New Version: 1.0.10

### DISCUSSION THREAD

**Discussion by ndortega** [OxygenFramework/Oxygen.jl#55] [Announcements]
Hey everybody,

In this latest patch, I focused on fleshing out the official documentation and adding some cool new features centered around the new `router()` function. This new function was modeled after the [APIRouter](https://fastapi.tiangolo.com/tutorial/bigger-applications/#apirouter) from fastapi but goes a bit further than the standard features. 

Guide:
-  https://ndortega.github.io/Oxygen.jl/stable/guide/

High-level Router features
- Easily share path prefix's between endpoints
- Tag routes ( used to visually group operations in the generated API documentation )
- Repeat Actions ( repeatedly call an endpoint on a set interval )

Read more about it here:
- https://github.com/ndortega/Oxygen.jl#routers
- https://github.com/ndortega/Oxygen.jl#repeat-actions

**Comment by HiperMaximus** [comment]
Pretty exicted about this new version!
